cmd/backup-manager/app/backup/backup.go

```diff
@@ -112,13 +115,39 @@ func (bo *Options) backupData(
 		go bo.updateProgressFromFile(progressCtx.Done(), backup, progressFile, progressStep, statusUpdater)
 	}
 
+	if backup.Spec.Mode == v1alpha1.BackupModeSnapshot && !bo.isBRCanContinueRunByCheckpoint() {
```

Comment:
Can we check whether the backup is restarted? I worry that we may delete valid backup data. I think we shouldn't clean backup data at first time. For example, user starts a backup with `retain` policy, then user deletes that backup. If user starts a backup with same name, we deletes valid backup data.

Reply:
ok, i think this commit can check this and only when backup is already running status, we will clean
https://github.com/pingcap/tidb-operator/pull/4883/commits/ffc46ec7178acc6fd22a7801c69ec1d2d99b6c4a

Reply:
i add running status check in https://github.com/pingcap/tidb-operator/pull/4883/commits/acecff3b4d6853b7f5a37e963cd53533aeaed370, PTAL

Reply:
after a discuss, i use `!scheduled` as the judging condition and add `restart` condition as a record.